openapi/mx_platform_api_beta.yml

```diff
@@ -578,15 +591,27 @@ components:
           example: false
           nullable: true
           type: boolean
+        is_subscription:
+          example:
```

Comment:
I noticed the example value was missing here.

Reply:
fixed.